code/src/components/SpinningLoader.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import styled from 'styled-components'
```

Comment:
We see that you have styled component-styling in a separate folder for a few components, but here you have the styling inside the component - maybe would be more consistent to do them all the same way? 
By the way - this loader is BITCHIN'! (that is a good thing) 👍 🚀 🥇